executable/mach_o.ksy

```diff
@@ -1246,3 +1282,89 @@ types:
             type: blob_index
             repeat: expr
             repeat-expr: count
+  build_version_command:
+    seq:
+      - id: platform
+        type: u4
+      - id: minos
+        type: u4
+      - id: sdk
+        type: u4
+      - id: ntools
+        type: u4
+      - id: tools
+        type: build_tool_version
+        repeat: expr
+        repeat-expr: ntools
+    types:
+      build_tool_version:
+        seq:
+          - id: tool
+            type: u4
+          - id: version
+            type: u4
+  segment_command:
+    seq:
+      - id: segname
+        type: str
+        size: 16
+        pad-right: 0
+        encoding: ascii
+      - id: vmaddr
+        type: u4
+      - id: vmsize
+        type: u4
+      - id: fileoff
+        type: u4
+      - id: filesize
+        type: u4
+      - id: maxprot
+        type: vm_prot
+      - id: initprot
+        type: vm_prot
+      - id: nsects
+        type: u4
+      - id: flags
+        type: u4
+      - id: sections
+        type: section
+        repeat: expr
+        repeat-expr: nsects
+    types:
+      section:
+        seq:
+          - id: sect_name
+            -orig-id: sectname
+            size: 16
+            type: str
+            pad-right: 0
+            encoding: ascii
+          - id: seg_name
+            -orig-id: segname
+            size: 16
+            type: str
+            pad-right: 0
+            encoding: ascii
+          - id: addr
+            type: u4
+          - id: size
+            type: u4
+          - id: offset
+            type: u4
+          - id: align
+            type: u4
+          - id: reloff
+            type: u4
+          - id: nreloc
+            type: u4
+          - id: flags
+            type: u4
+          - id: reserved1
+            type: u4
+          - id: reserved2
+            type: u4
+        instances:
+          data:
+            io: _root._io
+            pos: offset
+            size: size
```

Comment:
This file lacks end-of-file new line. Please re-add it.

Reply:
re-added, thanks for catching that.